cpp/src/io/utilities/column_buffer.hpp

```diff
@@ -192,7 +196,7 @@ class gather_column_buffer : public column_buffer_base<gather_column_buffer> {
     create(_size, stream, mr);
   }
 
-  void allocate_strings_data(rmm::cuda_stream_view stream);
+  void allocate_strings_data(rmm::cuda_stream_view stream, bool memset_data = true);
```

Comment:
Don't think this is being used.

```suggestion
  void allocate_strings_data(rmm::cuda_stream_view stream);
```

Reply:
I think I have to include the bool memset_data since the base class function has that argument right

Reply:
I don't see this function in the base class. Maybe I missed it, can you please confirm?

Reply:
In the create_with_mask function within the string case, it calls the allocate_strings_data function which can be either the inline_column_buffer or gather_column_function implementations. Because of this, wouldn't I have to make the arguments of both functions the same?